wordpress-mobile/WordPress-iOS
# Update custom icons

This PR updates the custom icons provided by the app.

| Before | After |
|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-08 at 19 11 37](https://user-images.githubusercontent.com/4780/114076859-0ff70100-989f-11eb-83d2-ae4f0d277cf0.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-08 at 19 07 48](https://user-images.githubusercontent.com/4780/114076868-11c0c480-989f-11eb-9f06-702748a90630.png) |

Most of the old icons have now been removed, so they are mapped over to the equivalent new icon when you launch the app:

<img width="337" alt="Screen Shot 2021-04-06 at 12 09 51 PM" src="https://user-images.githubusercontent.com/4780/114077029-3fa60900-989f-11eb-923f-68462a28b7b0.png">

The only exception to this is Pride – the existing icons are still bundled with the app (but hidden in the UI) so if you were using that 

**To test**

- Build and run develop, and choose a custom icon from Me > App Settings > App Icon
- Build and run this branch and check that the icon automatically switches to the equivalent, shown in the table above. Check in App Settings and ensure the correct icon is ticked (try this for some other icons too).
- Ensure you can switch between the new icons in the app.
- Try upgrading from the pride icon and ensure you still see that icon on the homescreen after the upgrade (Spectrum will be selected in the UI as it's the closest equivalent, but I can remove this mapping if it's confusing).

## Regression Notes

1. Potential unintended areas of impact

Nothing functional, just icon changes.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I tested the migration from each old icon type

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
